python/tank/folder/configuration.py

```diff
@@ -265,13 +265,63 @@ def _load_schema(self, schema_config_path):
                 )
 
             project_obj = Project.create(self._tk, project_folder, metadata)
+            ### Squeeze | Modification Begin
+            if not self._is_project_active(project_obj):
+                continue
+            ### Squeeze | Modification End
 
             # store it in our lookup tables
             self._entity_nodes_by_type["Project"].append(project_obj)
 
             # recursively process the rest
             self._process_config_r(project_obj, project_folder)
 
+    ### Squeeze | Modification Begin
+    def _is_project_active(self, project_obj):
```

Comment:
I would also change the name of the function to `_is_project_folder_template_active` to avoid confusion ;)

maybe the `project_obj` -> `project_folder_template_obj` too